test/unit/vm/disposable_test.dart

```diff
@@ -67,6 +71,23 @@ void main() {
       });
     });
 
+    group('manageDisposer', () {
+      test(
+          'should call callback and accept null return value'
+          'when parent is disposed', () async {
+        thing.testManageDisposer(expectAsync(() => null, count: 1) as Disposer);
```

Comment:
1 is the default value for the `count` parameter. You could just drop the param if you like.